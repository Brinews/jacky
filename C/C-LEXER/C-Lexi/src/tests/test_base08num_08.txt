
     0000001x08
