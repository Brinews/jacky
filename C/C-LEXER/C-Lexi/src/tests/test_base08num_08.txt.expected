{0 , NUM , 2}
{0 , NUM , 2}
{0 , NUM , 2}
{0 , NUM , 2}
{0 , NUM , 2}
{0 , NUM , 2}
{1x08 , BASE08NUM , 2}
{ , END_OF_FILE , 4}
